substrate/scripts/run_all_benchmarks.sh

```diff
@@ -97,46 +95,50 @@ rm -f $ERR_FILE
 # Benchmark each pallet.
 for PALLET in "${PALLETS[@]}"; do
   # If `-p` is used, skip benchmarks until the start pallet.
-  if [ ! -z "$start_pallet" ] && [ "$start_pallet" != "$PALLET" ]
-  then
+  if [ ! -z "$start_pallet" ] && [ "$start_pallet" != "$PALLET" ]; then
     echo "[+] Skipping ${PALLET}..."
     continue
   else
     unset start_pallet
   fi
 
-  FOLDER="$(echo "${PALLET#*_}" | tr '_' '-')";
+  FOLDER="$(echo "${PALLET#*_}" | tr '_' '-')"
   WEIGHT_FILE="./frame/${FOLDER}/src/weights.rs"
 
+  TEMPLATE_FILE_NAME="frame-weight-template.hbs"
+  if [ $(cargo metadata --locked --format-version 1 --no-deps | jq --arg pallet "${PALLET//_/-}" -r '.packages[] | select(.name == $pallet) | .dependencies | any(.name == "polkadot-sdk-frame")') = true ]; then
+    TEMPLATE_FILE_NAME="frame-umbrella-weight-template.hbs"
+  fi
+  TEMPLATE_FILE="./.maintain/${TEMPLATE_FILE_NAME}"
+
   # Special handling of custom weight paths.
-  if [ "$PALLET" == "frame_system_extensions" ] || [ "$PALLET" == "frame-system-extensions" ]
-  then
+  if [ "$PALLET" == "frame_system_extensions" ] || [ "$PALLET" == "frame-system-extensions" ]; then
     WEIGHT_FILE="./frame/system/src/extensions/weights.rs"
-  elif [ "$PALLET" == "pallet_asset_conversion_tx_payment" ] || [ "$PALLET" == "pallet-asset-conversion-tx-payment" ]
-  then
+  elif [ "$PALLET" == "pallet_asset_conversion_tx_payment" ] || [ "$PALLET" == "pallet-asset-conversion-tx-payment" ]; then
```

Comment:
What is happening here? I am generally not a fan of formatting changes when they are not enforced by CI.  
One dev changes it to a specific way and the next dev changes it back.

It also makes it much slower to review, since i now would need to check every line that was touched.

Reply:
I see, went back to previous formatting in https://github.com/paritytech/polkadot-sdk/pull/5995/commits/d46029526908a15a9d76c738afc92168643cea1b